anvil/src/cmd.rs

```diff
@@ -468,6 +468,20 @@ pub struct AnvilEvmArgs {
     pub steps_tracing: bool,
 }
 
+impl AnvilEvmArgs {
+    pub fn resolve_rpc_alias(&mut self) {
+        if let Some(fork_url) = &self.fork_url {
+            if fork_url.url.starts_with("http://") || fork_url.url.starts_with("https://") {
+                return
+            }
+            let config = Config::load();
+            if let Some(Ok(url)) = config.rpc_endpoints.resolved().get(&fork_url.url) {
+                self.fork_url = Some(ForkUrl { url: url.to_string(), block: fork_url.block });
+            }
```

Comment:
looks like we're actually missing this as a useful function in Config directly.

please take a look at `Config::get_rpc_url`, it would be great to extract this 
https://github.com/foundry-rs/foundry/blob/main/config/src/lib.rs#L774-L774
to a standalone function instead.
like `get_rpc_url_with_alias(alias:&str)`

Reply:
I factored out the alias lookup as suggested.
Please lmk if this is not what you had in mind: https://github.com/foundry-rs/foundry/pull/4384/commits/a23d5fcdc2054e85ebb30039d6c9ee36f562315d